Lib/http/cookies.py

```diff
@@ -293,6 +293,17 @@ def __init__(self):
         for key in self._reserved:
             dict.__setitem__(self, key, "")
 
+    def __class_getitem__(cls, type):
+        """Provide minimal support for using this class as generic
+        (for example in type annotations).
+
+        See PEP 484 and PEP 560 for more details. For example, `Morsel[T]`
+        is a valid expression at runtime (type argument `T` indicates the
+        type used for mode). Note, no type checking happens at runtime, but
```

Comment:
I think this is wrong. This type should represent value type for a key-value pair held. But probably it is best if someone familiar with this code will take a look.